src/web/lib/contributions.test.ts

```diff
@@ -0,0 +1,67 @@
+import {
+    getLastOneOffContributionDate,
+    ONE_OFF_CONTRIBUTION_DATE_COOKIE,
+    SUPPORT_ONE_OFF_CONTRIBUTION_COOKIE,
+} from './contributions';
+
+const setCookie = (name: string, value: string) => {
```

Comment:
Arguably, this test is purer because it does this itself however, there is a `setCookie` function in the `cookie` lib

Reply:
I've noticed DCR's helper to set cookies is identical (ported over) to Frontend's, so arguably the more reliable way of setting test cookies will be using the same logic the app uses. As that logic is the same on Frontend and DCR, I've refactored to use the DCR helper. Thanks for the tip!